osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -70,6 +67,9 @@ private void load(OsuConfigManager osuConfig, FrameworkConfigManager config)
                 },
             };
 
+            windowMode = config.GetBindable<WindowMode>(FrameworkSetting.WindowMode);
+            windowMode.BindValueChanged(mode => confineMouseModeSetting.Alpha = mode.NewValue == WindowMode.Fullscreen ? 0 : 1);
```

Comment:
while we're here, this probably requires a `true`, else the initial state may be incorrect.

```suggestion
            windowMode.BindValueChanged(mode => confineMouseModeSetting.Alpha = mode.NewValue == WindowMode.Fullscreen ? 0 : 1, true);
```